soroban-sdk/src/testutils.rs

```diff
@@ -21,7 +21,7 @@ pub trait ContractFunctionSet {
 }
 
 #[doc(inline)]
-pub use crate::env::internal::LedgerInfo;
+pub use crate::env::testutils::*;
```

Comment:
I think this was the original intent for the env::testutils mod.

Reply:
Actually, abandoning that env::testutils thing. Both LedgerInfo and Budget are exported in different ways in this mod and its better.